src/utils/simplified-account/settings/stripe-details/tasks.test.js

```diff
@@ -74,41 +68,32 @@ describe('friendlyStripeTasks', () => {
 
   it('should format paths correctly for task', () => {
     const accountWithSingleTask = {
-      ...account,
-      connectorGatewayAccountStripeProgress: {
-        director: true
-      }
+      director: true
     }
 
-    const result = friendlyStripeTasks(accountWithSingleTask, service)
+    const result = friendlyStripeTasks(accountWithSingleTask, account.type, service.externalId)
 
     expect(result[0].href).to.equal(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.stripeDetails.director, SERVICE_EXTERNAL_ID, ACCOUNT_TYPE))
   })
 
   it('should handle status values correctly', () => {
     const accountWithMixedStatus = {
-      ...account,
-      connectorGatewayAccountStripeProgress: {
-        vatNumber: true,
-        companyNumber: false,
-        governmentEntityDocument: false
-      }
+      vatNumber: true,
+      companyNumber: false,
+      governmentEntityDocument: false
     }
 
-    const result = friendlyStripeTasks(accountWithMixedStatus, service)
+    const result = friendlyStripeTasks(accountWithMixedStatus, account.type, service.externalId)
 
     expect(result[0].status).to.equal(COMPLETED_CANNOT_START) // eslint-disable-line
     expect(result[1].status).to.equal(NOT_STARTED) // eslint-disable-line
     expect(result[2].status).to.equal(CANNOT_START)
   })
 
   it('should handle empty progress object', () => {
-    const accountWithEmptyProgress = {
-      ...account,
-      connectorGatewayAccountStripeProgress: {}
-    }
+    const accountWithEmptyProgress = {}
```

Comment:
I guess this is really just `emptyProgress` since the account is no longer part of it. Same for the other `accountWith...` ones